### DISCUSSION TITLE
Gestione altri documenti

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#507] [Richieste di aiuto] [answered]
Buon pomeriggio, volevo sapere se fosse possibile, in qualche modo, implementare l'inserimento di altri tipi di documenti oltre a quelli già previsti (programmi, relazioni, 15 maggio).
Grazie, 
Saluti.

**Comment by trinko** [comment, marked as answer]
In futura sarà possibile i documenti con una procedura simile a quella per gli avvisi e circolari.
Spero di poterlo fare per il prossimo anno scolastico
